sockeye/output_handler.py

```diff
@@ -48,11 +50,15 @@ class OutputHandler:
     Abstract output handler interface
     """
 
-    def handle(self, t_input: sockeye.inference.TranslatorInput, t_output: sockeye.inference.TranslatorOutput):
+    def handle(self,
```

Comment:
Right now TranslationOutput is a NamedTuple and I like the idea of translator input/output objects being atomic/read-only.  The subtle issue is that creating the TranslationOutput object is part of translation and we want to record the total time of all method calls for translation.  In fact, looking at the code again I would consider moving the "tic" call to before creating TranslatorInput as well so we're measuring the total time from input string to output string.  Thoughts?